files/pnpm-audit/daily-pnpm-audit.sh

```
@@ -0,0 +1,120 @@
+#!/bin/bash
+set -e
+
+# Required env vars
+PROJECT_DIR="${AUDIT_PROJECT_DIR:?AUDIT_PROJECT_DIR not set}"
+SLACK_WEBHOOK="${AUDIT_SLACK_WEBHOOK:?AUDIT_SLACK_WEBHOOK not set}"
+
+# Ensure PATH includes homebrew and node binaries for cron/launchd environment
+export PATH="/opt/homebrew/bin:/usr/local/bin:$HOME/.nvm/versions/node/$(ls $HOME/.nvm/versions/node 2>/dev/null | tail -1)/bin:$PATH"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)